guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend_types.md

```diff
@@ -35,7 +35,7 @@ Modules with a *soft dependency* on another module can function properly without
 
 * The module extends another module's {% glossarytooltip 73ab5daa-5857-4039-97df-11269b626134 %}layout{% endglossarytooltip %}.
 
-<div class="bs-callout bs-callout-warning" id="warning">
+<div class="bs-callout bs-callout-warning" markdown="1">
```

Comment:
This is called out as a warning, but in the actual content it says "Note:". This seems incorrect.

Reply:
This issue is out of scope in this PR.